exercises/C-comparison-operators/exercise.js

```diff
@@ -5,16 +5,16 @@
   The variables should have values that match the expected results.
 */
 
-var studentCount = 16;
-var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+const studentCount = 16;
+const mentorCount = 9;
+const moreStudentsThanMentors = true; // finish this statement
 
-var roomMaxCapacity = 25;
-var enoughSpaceInRoom; // finish this statement
+const roomMaxCapacity = 25;
+const enoughSpaceInRoom = true; // finish this statement
```

Comment:
Maybe you could use the previous variables with comparison operators to replace 'true'.